src/Streamly/Internal/LZ4.hs

```diff
@@ -130,6 +136,49 @@ lz4_MAX_OUTPUT_SIZE :: CInt
 lz4_MAX_OUTPUT_SIZE =
     min (unsafePerformIO $ c_compressBound lz4_MAX_INPUT_SIZE) maxBound
 
+--------------------------------------------------------------------------------
+-- Conversion helpers
+--------------------------------------------------------------------------------
+
+{-# INLINE cIntToInt #-}
+cIntToInt :: CInt -> Int
+cIntToInt = fromIntegral
+
+{-# INLINE intToCInt #-}
+intToCInt :: Int -> CInt
+intToCInt = fromIntegral
```

Comment:
We can name it as `unsafeIntToCInt`. This can lose information and should be used with care.